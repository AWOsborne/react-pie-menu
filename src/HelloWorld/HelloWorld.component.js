import React from 'react';
import styled from 'styled-components';

const Content = styled.p`
  font-size: large;
`;

const HelloWorld = () => {
  console.log('Hello World!'); // eslint-disable-line no-console
  return (
    <Content>
       To get started, edit <code>src/HelloWorld/HelloWorld.component.js</code> and save to reload.
    </Content>
  );
};

export default HelloWorld;
