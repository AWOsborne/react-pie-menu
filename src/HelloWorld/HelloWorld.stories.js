import React from 'react';
import { storiesOf } from '@storybook/react';

import App from 'stories/App';
import index, { HelloWorld } from 'dist';

storiesOf('HelloWorld', module)
  .add('with App', () => (
    <App>
      <HelloWorld />
    </App>
  ))
  .add('default import', index());
