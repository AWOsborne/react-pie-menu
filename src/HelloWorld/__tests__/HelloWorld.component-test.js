import React from 'react';
import { shallow } from 'enzyme';

import HelloWorld from '../HelloWorld.component';

test('component <HelloWorld /> should render correctly', () => {
  const wrapper = shallow(<HelloWorld />);
  expect(wrapper).toMatchSnapshot();
});
