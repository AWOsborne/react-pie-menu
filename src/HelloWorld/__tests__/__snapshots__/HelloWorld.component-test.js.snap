// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <HelloWorld /> should render correctly 1`] = `
<HelloWorldcomponent__Content>
  To get started, edit 
  <code>
    src/HelloWorld/HelloWorld.component.js
  </code>
   and save to reload.
</HelloWorldcomponent__Content>
`;
